sdk/tables/Azure.Data.Tables/samples/README.md

```diff
@@ -20,7 +20,7 @@ Description of Azure Tables. Covers following functions:
 * Authenticating the client
 
 You can find samples for each of this main functions below.
-To get started you'll need an Azure Tables endpoint and credentials. See Azure Tables Client Library [Readme](../readme.md) for more information and instructions.
+To get started you'll need an Azure Tables endpoint and credentials. See Azure Tables Client Library [Readme](../README.md) for more information and instructions.
```

Comment:
nit:  I think you need an absolute link there due to the way that samples are published.  I recall this being an issue elsewhere.